### PR TITLE
Implemented a login form for users or the admin to sign into, which r…

### PR DESCRIPTION
**Pull request by leepatrickworrall**
…edirects on success, and displays an error message on failure.

### CODE REVIEW

**Comment by leepatrickworrall** [comment]
Most of the changes are file renames, and also you'll need to ignore the admin/css files.

File: ketchup/main.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def determine_end_and_start_date():
 # TODO: Re-add the "Assign both a limit to how many episodes a day you watch,
 #       as well as how many days a week you decide to watch.
 def run():
-    print("\nHello and welcome to AnimeKetchup "
+    print("\nHello and welcome to animeketchup "
```
Comment (chellygel) [inline ketchup/main.py:63]: i mean sure but why lol

File: website/animeketchup/static/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
+body {
+    background-color: #1a1a1a;
+    font-family: "Comic Sans MS", serif;
```
Comment (chellygel) [inline website/animeketchup/static/css/styles.css:3]: l m a o

**Review by chellygel — APPROVED** [review]
LGTM, great work!

File: ketchup/main.py (hunk 1 above)
Comment (leepatrickworrall) [reply ketchup/main.py:63]: PyCharm refactoring for you. Changes any instance of "AnimeKetchup" to "animeketchup" instead. It wasn't me being picky this time around haha.

File: website/animeketchup/static/css/styles.css (hunk 2 above)
Comment (leepatrickworrall) [reply website/animeketchup/static/css/styles.css:3]: For the _style._

**Pull request merged**